app/models/organization.rb

```diff
@@ -76,4 +77,19 @@ def primary_contact_is_member
     errors.add(:primary_contact, :invalid)
   end
 
+  def us_required_fields
+    return if !country_is_us?
+    return if postal_code.present? && state.present?
+    if postal_code.blank?
+      errors.add(:postal_code, :required_for_us)
+    end
+    if state.blank?
+      errors.add(:state, :required_for_us)
+    end
+  end
+
+  def country_is_us?
+    us_id = Country.find_by(name: 'United States').try(:id)
+    us_id.present? && country_id == us_id
+  end
 end
```

Comment:
Curious if there is a better way to arrange the logic here

Reply:
I'd been hoping to not need a custom validation, and just validate presence of state and postal_code if `country_is_us?` but the private method isn't available in a conditional validation.